ext-lib-rx-extensions/src/main/java/com/appunite/ext_lib_rx_extensions/ResponseOrErrorExtensions.kt

```diff
@@ -0,0 +1,38 @@
+package com.appunite.ext_lib_rx_extensions
+
+import com.appunite.rx.ResponseOrError
+import rx.Observable
+
+
+fun <T> Observable<ResponseOrError<T>>.onlySuccess(): Observable<T> {
+    return this.compose(ResponseOrError.onlySuccess())
+}
+
+fun <T> Observable<ResponseOrError<T>>.onlyError(): Observable<Throwable> {
+    return this.compose(ResponseOrError.onlyError())
+}
+
+fun <T> Observable<T>.toResponseOrError(): Observable<ResponseOrError<T>> {
```

Comment:
actually ResponseOrError should never be used look on Try from https://github.com/MarioAriasC/funKTionale

Reply:
sure but this lib is only for our legacy projects (95% of them)